### PR TITLE
Added db seeding

### PR DESCRIPTION
**Pull request by SilGosker**
Added Seed class and method, added Arjan as admin to the DB and some profile settings

### CODE REVIEW

File: Data/Migrations/20221201110602_initial seed.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    { -10, "Experience", null, null },
+                    { -9, "Goals", null, null },
+                    { -8, "Education", null, null },
+                    { -7, "Description", null, null }
+                });
+
+            migrationBuilder.InsertData(
+                table: "Users",
+                columns: new[] { "Id", "Email", "Hash", "ModifyDate", "Name", "Salt", "TimeZoneId", "UserType" },
+                values: new object[] { 1, "artimmerman@landstede.nl", "$2a$11$A1PAL2tcek6yMqg8VVVzauteuOFGnD1S4DoqlPP/Hf9ulBwfJTS8y", null, "Arjan Timmerman", "$2a$11$FIhli04K3CDnTp4ObcYIb.", "Africa/Abidjan", 4 });
```
Comment (SophiaH67) [inline Data/Migrations/20221201110602_initial seed.cs:27]: Maybe add a comment with the original unhashed password?

File: Data/Migrations/ContextModelSnapshot.cs [hunk 2]
```diff
@@ -329,6 +367,19 @@ protected override void BuildModel(ModelBuilder modelBuilder)
...
 
                     b.ToTable("Users");
+
+                    b.HasData(
+                        new
+                        {
+                            Id = 1,
+                            CreateDate = new DateTime(2022, 12, 1, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            Email = "artimmerman@landstede.nl",
+                            Hash = "$2a$11$A1PAL2tcek6yMqg8VVVzauteuOFGnD1S4DoqlPP/Hf9ulBwfJTS8y",
```
Comment (SophiaH67) [inline Data/Migrations/ContextModelSnapshot.cs:377]: Same here

File: Data/Migrations/SeedConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            });
+    }
+    public static void Seed(ModelBuilder builder)
+    {
+        builder.Entity<User>().HasData(new User()
+            {
+                Email = "artimmerman@landstede.nl",
+                CreateDate = BaseDate,
+                Hash = "$2a$11$A1PAL2tcek6yMqg8VVVzauteuOFGnD1S4DoqlPP/Hf9ulBwfJTS8y",
+                Salt = "$2a$11$FIhli04K3CDnTp4ObcYIb.", //testen as password
```
Comment (SophiaH67) [inline Data/Migrations/SeedConfiguration.cs]: "testen" is the password? If so, why is the comment on the salt?

**Review by SophiaH67 — CHANGES_REQUESTED** [review]

File: Data/Migrations/SeedConfiguration.cs (hunk 3 above)
Comment (HUSKUT) [reply Data/Migrations/SeedConfiguration.cs]: very salty comment

Comment (SilGosker) [reply Data/Migrations/SeedConfiguration.cs]: You're right, fixed it

**Review by SophiaH67 — APPROVED** [review]

**Review by HUSKUT — APPROVED** [review]

**Comment by HUSKUT** [comment]
LGTM

**Pull request merged**